$OpenBSD: patch-t_test-lib_sh,v 1.2 2014/03/10 20:56:53 pascal Exp $

Force usage of GNU tar in the test suite: 't5004-archive-corner-cases.sh'
wants it badly.

--- t/test-lib.sh.orig	Fri Feb 14 22:57:09 2014
+++ t/test-lib.sh	Sun Feb 16 18:13:37 2014
@@ -51,6 +51,10 @@ fi
 . "$GIT_BUILD_DIR"/GIT-BUILD-OPTIONS
 export PERL_PATH SHELL_PATH
 
+if [ -n "${TEST_TAR}" ]; then
+	export TAR=${TEST_TAR}
+fi
+
 # if --tee was passed, write the output not only to the terminal, but
 # additionally to the file test-results/$BASENAME.out, too.
 case "$GIT_TEST_TEE_STARTED, $* " in
